insights/specs/insights_archive.py

```diff
@@ -15,6 +15,7 @@ class InsightsArchiveSpecs(Specs):
     bios_uuid = simple_file("insights_commands/dmidecode_-s_system-uuid")
     blkid = simple_file("insights_commands/blkid_-c_.dev.null")
     brctl_show = simple_file("insights_commands/brctl_show")
+    cdc_wdm = simple_file("/sys/bus/usb/drivers/cdc_wdm/module/refcnt")
```

Comment:
@vishwanathjadhav this is not necessary for a `simple_file`.  It will use the setting from default.py for archives.

Reply:
Done